hack/coverage.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Does it possible that you forgot to run `chmod +x` on the script, because travis again fails on it.